kit/ChildSession.cpp

```diff
@@ -3146,6 +3233,33 @@ void ChildSession::loKitCallback(const int type, const std::string& payload)
         break;
     case LOK_CALLBACK_EXPORT_FILE:
     {
+        bool isAbort = payload == "ABORT";
+        bool isPending = payload == "PENDING";
```

Comment:
Having enum for these would be best, but I'm not sure I understand when we expect "pending" and when we expect "abort".

Reply:
This comes from the dialog execution result:
- ok/export -> PENDING
- cancel -> ABORT
- error during export -> ERROR
- export finished with success -> URL in the payload